samples/ComputerVision/AnalyzeImage/Program.cs

```diff
@@ -0,0 +1,226 @@
+﻿using Microsoft.Azure.CognitiveServices.Vision.ComputerVision;
+using Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Models;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace AnalyzeImage
+{
+    class AnalyzeImage
+    {
+        private const string subscriptionKey = "<your training key here>"; //Insert your Cognitive Service subscription key here
+
+        // Specify the features to return
+        private static readonly List<VisualFeatureTypes> features =
+            new List<VisualFeatureTypes>()
+            {
+            VisualFeatureTypes.Categories, VisualFeatureTypes.Description,
+            VisualFeatureTypes.Faces, VisualFeatureTypes.ImageType,
+            VisualFeatureTypes.Tags, VisualFeatureTypes.Adult,
+            VisualFeatureTypes.Color, VisualFeatureTypes.Brands,
+            VisualFeatureTypes.Objects
+            };
+
+        static void Main(string[] args)
+        {
+            ComputerVisionClient computerVision = new ComputerVisionClient(new ApiKeyServiceClientCredentials(subscriptionKey));
+
+            // You must use the same Azure region that you generated your subscription keys for.  Free trial subscription keys are generated in the westus region. 
+            computerVision.Endpoint = "https://westus.api.cognitive.microsoft.com";
+
+            string localImagePath = @"Images\celebrities.jpg"; // See this repo's readme.md for info on how to get these images. Alternatively, you can just set the path to any appropriate image on your machine.
+            string remoteImageUrl = "https://github.com/Azure-Samples/cognitive-services-sample-data-files/raw/master/ComputerVision/Images/landmarks.jpg";
+
+            Console.WriteLine("Images being analyzed ...");
+            var t1 = AnalyzeFromUrlAsync(computerVision, remoteImageUrl);
```

Comment:
I recommend to create two public static functions similar to https://github.com/Azure-Samples/cognitive-services-dotnet-sdk-samples/blob/master/samples/TemplateMultiple/SecondSample/Sample.cs
So that the samples will be auto-testable.